### ISSUE TITLE
v1.2.5 breaks Promise.resolve and Promise.reject

### ISSUE THREAD

**Issue by srijken** [webcomponents/webcomponentsjs#983]
Promise.resolve and Promise.reject no long work as of v1.2.5, and this breaks IE11 compatibility.

**Comment by web-padawan** [comment]
Confirmed, and BTW the polyfill replacement have been at least minor bump for 1.x

```
Windows 10 IE 11         ✖ Test Suite Initialization
  Object doesn't support property or method 'resolve'
    <unknown> at   Anonymous function at /components/webcomponentsjs/webcomponents-lite.js:217:0
    <unknown> at          Global code at /components/webcomponentsjs/webcomponents-lite.js:10:0
```

**Comment by web-padawan** [comment]
See also https://github.com/webcomponents/webcomponentsjs/pull/977/files/4a1731fac34f3688fb9190f8504944d7834e5f2f#r210449813

**Comment by srijken** [comment]
it's also unfortunate that 1.2.5 was pushed with failing tests: https://travis-ci.org/webcomponents/webcomponentsjs/builds/418483536#L2862-L2864

**Comment by azakus** [comment]
Argh, sorry for this. Closure Compiler's renaming changed with a newer version.

**Comment by azakus** [comment]
Publishing 1.2.6 with #984 and #986 for `Promise.prototype.then`.

**Comment by azakus** [comment]
Postmortem on the bad release:
Saucelabs starting having issues running tests at around the time I published 1.2.5 (https://status.saucelabs.com/incidents/4mqv8wg521r0), and I didn't investigate the test failures because I assumed it was the same issue.
